include/acl_hal_mmd.h

```diff
@@ -33,6 +33,26 @@ typedef struct {
   unsigned long long size;  /* size of this memory */
 } aocl_mmd_memory_info_t;
 
+// Interface to simulator to describe streaming kernel arguments that are
+// excluded from the invocation image. Streaming arguments are passed to the
+// simulator by calling aocl_mmd_simulation_streaming_kernel_info(), before
+// writing the kernel invocation image containing non-streaming arguments.
+
+// The pointers are valid while aocl_mmd_simulation_streaming_kernel_info()
+// is being called only and shall not be used after the function returns.
+struct aocl_mmd_streaming_kernel_arg_info_t {
+  // unique identifier for the bus-functional model (BFM)
+  const char *name;
+  // argument value
+  const char *value;
```

Comment:
Ideally a byte array should use the pointer type
```suggestion
  const unsigned char *value;
```
but we must use the same type as in [cl_kernel](https://github.com/intel/fpga-runtime-for-opencl/blob/83a27ea2bf2822399591668975a26046de78a426/include/acl_types.h#L909).